### ISSUE TITLE
Error when hitting Refresh (F5) 

### ISSUE THREAD

**Issue by AndyHunt66** [AntonioSoler/bga-takaraisland#4]
I hit F5 at the end of my turn, when it was saying I could pay the hospital or buy a worker from the beach,

I got the following:
```
Javascript error:
During pageload
Cannot set property 'push' of undefined
TypeError: Cannot set property 'push' of undefined
at Object.onEnteringState (http://1.studio.boardgamearena.com:8081/data/themereleases/151226-1240/games/takaraisland/999999-9999//takaraisland.js?1491766267138:291:27)
at Object.completesetup (http://1.studio.boardgamearena.com:8081/data/themereleases/151226-1240/js/modules/layer/ly_studio.js:109:86658)
at http://en.1.studio.boardgamearena.com/takaraisland?table=38005&testuser=2303477:875:24
at _cc (http://1.studio.boardgamearena.com:8081/data/themereleases/151226-1240/js/dojoroot/dojo/dojo.js:15:11822)
at _37 (http://1.studio.boardgamearena.com:8081/data/themereleases/151226-1240/js/dojoroot/dojo/dojo.js:15:13422)
at http://1.studio.boardgamearena.com:8081/data/themereleases/151226-1240/js/dojoroot/dojo/dojo.js:15:13668
at _38 (http://1.studio.boardgamearena.com:8081/data/themereleases/151226-1240/js/dojoroot/dojo/dojo.js:15:13512)
at _80 (http://1.studio.boardgamearena.com:8081/data/themereleases/151226-1240/js/dojoroot/dojo/dojo.js:15:13593)
at _f4 (http://1.studio.boardgamearena.com:8081/data/themereleases/151226-1240/js/dojoroot/dojo/dojo.js:15:15359)
at HTMLScriptElement._10a (http://1.studio.boardgamearena.com:8081/data/themereleases/151226-1240/js/dojoroot/dojo/dojo.js:15:18265)
Script: 
Url: http://en.1.studio.boardgamearena.com/takaraisland?table=38005&testuser=2303477
U=2303477
Mozilla/5.0 (Windows NT 6.3; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/57.0.2987.133 Safari/537.3
```

**Comment by AndyHunt66** [comment]
The page reloaded ok though, apart from the message
`Application loading ...`
in a separate bar, underneath the "You can pay the hospital or recruit one worker on the beach, when done press:" bar

**Comment by AntonioSoler** [comment]
this should be fixed in 7d63d1901ea8287081be5e75983131a2f37c85c3 as now i don't use an array to store dojo connections for recruits
